gatsby-node.js

```diff
@@ -44,3 +44,19 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     })
   })
 }
+
+exports.onCreateWebpackConfig = ({
+  stage,
+  rules,
+  loaders,
+  plugins,
+  actions,
+}) => {
+  actions.setWebpackConfig({
+    resolve: {
+      alias: {
+        DocComponents: path.resolve(__dirname, 'src/doc-components/')
```

Comment:
I'm guessing this lets us resolve doc components for use in the `mdx` files?

Reply:
yes. makes it easier to import from mdx files without using relative paths